.gitignore

```diff
@@ -41,3 +41,6 @@ yarn-debug.log*
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+# Ignore Jetbrains IDE files
+/.idea
```

Comment:
Kept accidentally adding my .idea folder from RubyMine... I could make a separate PR for this but I hope this is fine too.